app/staking/page.tsx

```diff
@@ -267,9 +267,19 @@ export default function StakingPage() {
       <div>
         <Spacer height="20px" />
       </div>
-      <Text size="x-lg" font="proto_mono" className={styles.title}>
-        STAKING
-      </Text>
+      <div
+        style={{
+          display: "flex",
+          width: "100%",
+          flexDirection: "row",
```

Comment:
why are we wrapping text in a div and adding flex, direction, alignment to it. I just removed the whole component and the text is still being rendered fine. so why are we adding additional inline ?